server/src/main/java/org/apache/iotdb/db/qp/physical/sys/AlterTimeSeriesPlan.java

```diff
@@ -102,6 +103,15 @@ public String getAlias() {
     return Collections.singletonList(path);
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    List<String> ret = new ArrayList<>();
+    for (Path curPath : Collections.singletonList(path)) {
+      ret.add(curPath.getFullPath());
+    }
+    return ret;
```

Comment:
Just `Return Collections.singletonList(path.getFullPath());` and check other places, please.

Reply:
I'll change it. Thanks for your suggestion.